wordpress-mobile/WordPress-Aztec-Android
# Quote gets removed

### Fix #288 

Expanding a block to neighbouring same-type blocks should only happen if neighbours are at the same nesting level.

### Test
1. Start with the demo text
2. Tap the word inside blockquote and apply a bullet list
3. Switch to HTML
4. Switch back to visual
5. Notice the blockquote style is still there and a bullet list is nested inside it
